schemas/2015-08-01/Microsoft.Cache.json

```diff
@@ -0,0 +1,160 @@
+﻿{
+    "id": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.Cache.json",
```

Comment:
To expose this to template author, please add the reference to 2014-04-01-preview\deploymentTemplate.json and 2015-01-01\deploymentTemplate.json